html/views/overlays/interactive/admin.js

```diff
@@ -133,7 +132,7 @@ $('select#Skaters').change(function(e) {
 	team = $( 'option[value=' + v + ']', $t ).attr('data-team');
 	name = $( 'option[value=' + v + ']', $t ).attr('data-name');
 	tnam = WS.state['ScoreBoard.Team(' + team + ').AlternateName(overlay)'];
-	tnam = tnam ? tnam : WS.state['ScoreBoard.Team(' + team + ').Name'];
+	tnam = (tnam != "") ? tnam : WS.state['ScoreBoard.Team(' + team + ').Name'];
```

Comment:
Isn't this `null` if the AlternateName isn't set?

Reply:
Hmm, thought I'd already undone that one.